src/cow-react/modules/swap/containers/RowSlippage/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { useContext, useMemo } from 'react'
+import { Percent } from '@uniswap/sdk-core'
+import { ThemeContext } from 'styled-components/macro'
+
+import { PERCENTAGE_PRECISION } from 'constants/index'
+import { useToggleSettingsMenu } from 'state/application/hooks'
+import { formatSmart } from 'utils/format'
+import { RowSlippageContent } from '@cow/modules/swap/pure/Row/RowSlippageContent'
+import { RowCommonProps } from '@cow/modules/swap/pure/Row/typings'
+
+export interface RowSlippageProps extends Omit<RowCommonProps, 'showHelpers'> {
+  allowedSlippage: Percent
+  showSettingOnClick?: boolean
+}
+
+export function RowSlippage({
+  allowedSlippage,
+  fontSize = 13,
```

Comment:
To avoid deep stuffing of styles into the component, maybe will be better to use Provider instead

Reply:
You can ignore it, because it wasn't added by you

Reply:
👍